### ISSUE TITLE
feat: Add adopters website

### ISSUE THREAD

**Issue by mauromorales** [kairos-io/kairos#2555]
We want to have a page on kairos.io that celebrates our adopters. Some reasons for this:

- We want a single place where people coming to Kairos can see that it's being used on production/homelabs by real companies, organizations and individuals
- We want our adopters logo's to be showcased next to our partners

**Comment by ci-robbot** [comment]
Hello mauromorales, thank you for bringing up the idea of adding an adopters website to the kairos.io repository. However, before we can proceed, we need to ensure that the ticket meets the project's requirements for a new issue. Could you please provide a description of the issue and any relevant steps to reproduce (if it's a bug)? Additionally, please mention the versions of the relevant artifacts being used.

Once this information is provided, we will be able to properly evaluate and label the issue accordingly.

Thank you for your understanding, and remember that this is an experiment by @mudler and @jimmykarily to improve the issue management process.

**Comment by jimmykarily** [comment]
How about creating an issue template for this? Something like a form to apply to be included in the adopters. Then we advertise it and let people upload logos and urls so we can populate the list.

**Comment by mauromorales** [comment]
@jimmykarily indeed 🙌 ... we already have the form, and we're also cold contacting people from the community, hopefully we'll get a few to join.

https://github.com/kairos-io/community/blob/main/.github/ISSUE_TEMPLATE/adoption.yaml

**Comment by mauromorales** [comment]
we have an adopters section in the homepage now, since it is extensible but we don't have enough to move them into their own page, I'll close this
